AIDevGallery/Pages/AddModelPage.xaml

```diff
@@ -68,20 +68,23 @@
             </Grid>
         </Grid>
 
-        <ScrollViewer Grid.Row="1" Padding="24,0,24,0">
-            <ItemsRepeater
-                Grid.Row="1"
-                Margin="0,24,0,24"
-                ItemsSource="{x:Bind results}"
-                TabFocusNavigation="Local">
-                <ItemsRepeater.Layout>
-                    <UniformGridLayout MinColumnSpacing="16" MinRowSpacing="32" />
-                </ItemsRepeater.Layout>
-                <ItemsRepeater.ItemTemplate>
-                    <DataTemplate x:DataType="local:Result">
+        <ItemsView Grid.Row="1"
```

Comment:
This change will make the cards react on mouse hover. That shouldn't happen as it indicates that it would be an interactive element, which the card itself is not.

Reply:
reverted